.eslintrc.js

```diff
@@ -14,5 +14,12 @@ module.exports = {
     'ecmaVersion': 2018,
   },
   'rules': {
+    'func-style': [
+      "error",
+      "declaration",
```

Comment:
I think you want "expression" based on what I've seen of your style so far. (Then allowArrowFunctions is not necessary).

Reply:
Sorry, I see you switched everything to declarations. In that case maybe try removing allowArrowFunctions?

Reply:
Do I need to completely disallow arrow functions?

Reply:
If I am reading the docs right, removing "allowArrowFunctions" will not totally disallow arrow functions. It will just prevent you from using them when declaring new functions. Which is good, because you've decided to stick to `function foo() { }`, instead of `const foo = () => { }`.